lms/templates/error.html.jinja2

```diff
@@ -3,8 +3,10 @@
 {% set title = message %}
 
 {% block content %}
-  <img src="{{ 'lms:static/images/sad-annotation.svg'|static_path }}">
-  <p>
-    {{ message|safe }}
-  </p>
+  <main class="modal-content">
+    <img src="{{ 'lms:static/images/sad-annotation.svg'|static_path }}">
```

Comment:
The message is no longer centered horizontally underneath the logo. Is that intended? The same goes for the other places where this wrapper was added.

<img width="211" alt="Screenshot 2019-04-25 14 45 48" src="https://user-images.githubusercontent.com/2458/56740509-fbc8a080-6768-11e9-94a9-0d74e290ca59.png">

Reply:
Fixed, thanks. I duplicated a little CSS to fix it. The CSS in this project needs sorting, there isn't much of a system to it, it's not really maintainable. There's not much CSS or HTML so it wouldn't take too long but I don't want to do it for this PR.